cmd/janus/init.go

```diff
@@ -18,6 +21,7 @@ var (
 	globalConfig *config.Specification
 	statsClient  stats.Client
```

Comment:
Ah ok, I see now, you just declare them here and pass the way down to the dependencies on main.go